### DISCUSSION TITLE
friends - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#23] [Announcements]
# friends - 没有秃头基因的燊的博客

喜爱折腾的人 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/friends/

**Comment by wumingblog** [comment]
博主您好，请问考虑换友链吗？

**Reply by yzs020220** [reply]
我这边友链都是熟人，平时也不太喜欢社交，但还是感谢了 :smile_cat:
